examples/main.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/optimizely/go-sdk"
```

Comment:
what's the need?

Reply:
This was forced by linter since we are using `optimizely` keyword to access the package in the sample code.